runtime/moonbase/src/weights/pallet_proxy.rs

```diff
@@ -216,4 +216,9 @@ impl<T: frame_system::Config> pallet_proxy::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
+
+    fn poke_deposit() -> Weight {
+        // TODO: regenerate weights
+        <()>::poke_deposit()
```

Comment:
Lets try to generate this locally, just to be sure the benchmarks are not broken